Source/HarmonyPatches/Patches_PermissionFix/Patch_ForibiddenDrop.cs

```diff
@@ -1,21 +0,0 @@
-using System;
-using HarmonyLib;
-using RimWorld;
-using Verse;
-
-namespace PrisonLabor.HarmonyPatches.Patches_PermissionFix
```

Comment:
So everything what is dropped by prisoners is forbidden now?

Reply:
So far no, However it does happen sometimes (bills, meals, hauling are ok), it happens when striped, also the old one was really really slow.